docs/cgroups_in_storm.md

```diff
@@ -43,13 +43,24 @@ group storm {
        }
        cpu {
        }
+       memory {
+       }
+       cpuacct {
+       }
 }
 ```
 
 For a more detailed explanation of the format and configs for the cgconfig.conf file, please visit:
 
 https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Resource_Management_Guide/ch-Using_Control_Groups.html#The_cgconfig.conf_File
 
+To let storm manage the cgroups for indavidual workers you need to make sure that the resources you want to control are mounted under the same directory as in the example above.
+If they are not in the same directory the supervisor will throw an exception.
+
+The perm section needs to be configured so that the user the supervisor is running as can modify the group.
+
+If run as user is enabled so the supervisor spawns other processes as the user that launched the topology make sure that the permissions are such that indavidual users have read access but not write access.
```

Comment:
perhaps more clean 

if "run as user"is enabled so **that** the supervisor spawns other...